.circleci/config.yml

```diff
@@ -84,10 +84,6 @@ jobs:
 workflows:
   build:
     jobs:
-      - test:
-          name: test_rails-5.2_ruby-2.7.6
-          ruby: 2.7.6
-          gemfile: gemfiles/rails_5.2_ruby_2.7.6.gemfile
```

Comment:
removed the tests in 5.2 as discussed in teams, rails 5.2 doesn't support Zeitwerk autoloader thus is doesn't have the `to_prepare` block